l2-frontend/src/pages/ResultsParaclinic.vue

```diff
@@ -359,7 +359,7 @@
                 <div class="results-history" :class="embedded && 'results-history-embedded'" slot="dropdown">
                   <ul>
                     <li v-for="r in research_history" :key="r.pk">
-                      Результат от {{ r.date }}
+                      от {{ r.date }}
```

Comment:
Можно, наверное, даже так
```suggestion
                      {{ r.date }}
```

Reply:
поправил